mtgjson5/providers/cardkingdom.py

```diff
@@ -81,9 +82,13 @@ def generate_today_price_dict(
 
             mtgjson_uuid = card_kingdom_id_to_mtgjson[card_id]
 
+            key = self.today_date
+            if detailed_mode:
```

Comment:
I don't agree with this change. AllPricesToday should be the same structure as AllPrices, except it only has one entry per op. This makes it easily changeable between the AllPrices file.